pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.38</version>
+        <version>4.52</version>
```

Comment:
Why not directly upgrade to latest parent (4.86) ?

Reply:
Make sense, thanks!